spec/support/factory_bot.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+    config.include FactoryBot::Syntax::Methods

```

Comment:
So `FactoryBot` is a dependency for `RSpec`?

Reply:
not really, it just makes thing easier for me

Reply:
Ok, will discuss it offline, I need more details on this